src/System.ComponentModel.Composition/tests/StringsTests.cs

```diff
@@ -0,0 +1,57 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Microsoft.Internal;
+using System.Threading;
+using System.Globalization;
+using Xunit;
+using Microsoft.VisualBasic;
+
+namespace System
+{
+    public class StringsTests
+    {
+        [Fact]
+        public void PropertiesAreInsyncWithResources()
+        {
+            var properties = GetStringProperties();
+
+            Assert.True(properties.Length > 0, "Expected to find at least one string property in Strings.cs.");
+
+            foreach (PropertyInfo property in properties)
```

Comment:
The problem in these types of situations is that it's impossible to tell _which_ (or how many) property failed.  Thankfully xunit provides a way for us to do that:
```
Assert.All(properties.Select(property => property.GetValue(null, (object[])null)), value => Assert.NotNull(value));
// or this, which may have a nicer "value" from the iterator:
Assert.All(properties, property => 
    {
        object value = property.GetValue(null, (object[])null);
        Assert.NotNull(value);
    });
```
...which is going to give you the index of the failure, at minimum.